terra_notebook_utils/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.0.0'
+__version__ = '0.1.1'
```

Comment:
Maybe add a comment here to not edit/bump this directly but to always run `make release`?

Reply:
Good idea! Will add.